vault/resource_pki_secret_backend_cert.go

```diff
@@ -56,6 +56,15 @@ func pkiSecretBackendCertResource() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"uri_sans": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "List of alternative URIs.",
+				ForceNew:    true,
```

Comment:
Should this really `ForceNew` if this field changes?

Reply:
Hi @tyrannosaurus-becks,
    To be quite honest, I lifted most of this code for a similar PR raised against CAs. https://github.com/terraform-providers/terraform-provider-vault/pull/373/files#diff-318e4d5fc8ea223f57928c837b6ea671R60. In that PR, it was set to ForceNew as well. I would think force new matches this matches the intent, as you probably need the uri set in the cert if your are taking the time to set it in the tf.